src/track/signal/SignalTrack.ts

```diff
@@ -116,6 +104,19 @@ export class SignalTrack<Model extends TrackModel = SignalTrackModel> extends Sh
         this.highlightPointer.mask = this;
         this.add(this.highlightPointer);
         
+
+        if (model.color != null) {
+            this.sharedState.signalColor = model.color;
+        }
+
+        if (model.autoScale != null) {
+            this.autoScale = model.autoScale;
+        }
+
+        if (model.scale != null) {
+            this.displayScale = model.scale;
+        }
+
```

Comment:
Is there a reason for moving this code?

Reply:
I don’t know how it was causing a bug. It comes from this commit: https://github.com/VALIS-software/valis-hpgv/commit/ee8a1104b35cc0337c65a4bd3a7b18bc59680ab7

Reply:
I see this now: "Prevent issue where yAxis is used before initialized"